common.sh

```diff
@@ -26,7 +26,9 @@ setup() {
 
     # Build yklua with JIT support.
     git clone https://github.com/ykjit/yklua
-    cd yklua
+    cd yklua/src
+    patch < ../../../../patches/clua_gettime
```

Comment:
One thing I've learned about patch: always specify a patch level.

Reply:
It works in this case. Do you have a suggested change?

Reply:
Just add a `-p`.

The problem is, if you don't give patch a `-p` it starts guessing where to apply the diff and if files of the same name exist in different places, it occasionally gets it wrong. It's bitten me a few times.

Although it works ok now, an future "sync with upstream lua" could cause fun!

I didn't have to search for long to find an example in the wild:
https://github.com/conda/conda-build/issues/4241

In this case I'd `cd` to the top level dir and use `-p0`.